packages/embark-ui/src/services/api.js

```diff
@@ -179,7 +179,7 @@ export function listenToChannel(credentials, channel) {
 }
 
 export function webSocketProcess(credentials, processName) {
-  return embarkAPI.webSocketProcess(credentials, `/process-logs/${processName}`);
```

Comment:
Was there a double-up of `process-logs` in the failing URLs before? For example, `embark-api/process-logs/process-logs/blockchain`. I guess I'm curious as to why there was no need for an update on the other end (ie in [`embark-api-client`](https://github.com/embark-framework/embark/blob/16cd84677e990d04fb06f49a3228225e198aed7e/packages/embark-api-client/src/index.js#L252)). At a glance, what you've done looks correct.

Reply:
There was. I've seen them too, but they didn't seem to cause errors in the network, which is why I wasn't sure if it was really an issue or just devtools just outputting certain protocol request in a different way...

Reply:
@emizzle yes, in the network tab of browser devtools they looked like `process-logs/[name]//process-logs/[name]`.

@PascalPrecht they didn't cause errors per se, but because of the bad URLs the process logs weren't  getting updates. You could toggle between the process log tabs and a fetch would result in the latest logs being loaded; but with the URL fix the updates come streaming in, which is the desired behavior.

Reply:
So really there were two bugs. One was related to the proxy on 55555 forwarding `/embark-api/` requests, and the other was the bad URLs. The effect of the second wasn't apparent until the first one was fixed.